ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1102,11 +1301,11 @@ export default {
           console.error('Octet error: bad party list size: ' + JSON.stringify(partyList));
           return;
         }
-        const uniqDict = {};
-        for (let i = 0; i < data.octetMarker.length; ++i) {
-          uniqDict[data.octetMarker[i]] = true;
-          if (!partyList.includes(data.octetMarker[i])) {
-            console.error('Octet error: could not find ' + data.octetMarker[i] + ' in ' + JSON.stringify(partyList));
+        const uniqDict: { [name: string]: boolean } = {};
+        for (const marker of data.octetMarker) {
+          uniqDict[marker] = true;
+          if (!partyList.includes(marker)) {
+            console.error('Octet error: could not find ' + marker + ' in ' + JSON.stringify(partyList));
```

Comment:
```suggestion
            console.error(`Octet error: could not find ${marker} in ${JSON.stringify(partyList)}`);
```
Is exactly 100 characters